packages/mailbox/src/mailbox.ts

```diff
@@ -0,0 +1,81 @@
+import { Users, Update, ThreadID, InboxListOptions, UserAuth, UserMessage, PrivateKey, Public, privateKeyFromString, PublicKey } from '@textile/hub';
+import { Identity, SpaceUser, GetAddressFromPublicKey } from '@spacehq/users';
+import { grpc } from '@improbable-eng/grpc-web';
+import ee from 'event-emitter';
+import { threadId } from 'worker_threads';
+
+export interface MailboxConfig {
+  textileHubAddress?: string;
+  usersInit?: (auth: UserAuth) => Users;
+}
+
+const DefaultTextileHubAddress = 'https://hub-dev-web.space.storage:3007';
+
+export class Mailbox {
+  constructor(private readonly user: SpaceUser, private readonly config: MailboxConfig = {}) {
+    this.config.textileHubAddress = this.config.textileHubAddress ?? DefaultTextileHubAddress;
+  }
+
+  //   public async ListInboxMessages(opts: InboxListOptions):[]UserMessage {
+  //     const messages = await this.client.listInboxMessages();
+  //     const inbox = [];
+  //     messages.forEach(async (msg) => {
+  //       inbox.push(await this.messageDecoder(msg));
+  //     });
+  // PrivateKey.fromString
+
+  //   };
+
+  public async SendMessage(from: string, to: string, body:Uint8Array): Promise<UserMessage> {
```

Comment:
Maybe we don't need the `from: string` parameter since we can get it from the `user: SpaceUser` passed into the constructor